README.md

```diff
@@ -4,13 +4,37 @@ This repository contains all the operational and user documentation for running
 
 ## Getting Started
 
-We have the [Open Data Hub](https://opendatahub.io/) applications deployed and running in a MOC ([Mass Open Cloud](https://massopen.cloud/)) cluster. All user documentation such as the login process for each of the applications deployed can be found [here](https://github.com/operate-first/odh-moc-support/tree/main/docs/user-docs).
+We have the [Open Data Hub](https://opendatahub.io/) applications deployed and running in a MOC ([Mass Open Cloud](https://massopen.cloud/)) cluster.
+
+ODH homepage on MOC is the [ODH Dashboard](https://odh-dashboard-opf-dashboard.apps.cnv.massopen.cloud/). From there you can access any of the provided application:
+
+https://odh-dashboard-opf-dashboard.apps.cnv.massopen.cloud/
+
+![ODh Dashboard](docs/assets/images/odh-dashboard.png)
+
+## Component status
+
+| Component               | Availability      | Authentication                                                                     |
+| ----------------------- | ----------------- | ---------------------------------------------------------------------------------- |
+| AI-Library              | ❌ (Not deployed) |                                                                                    |
+| Airflow                 | ❌ (Not deployed) |                                                                                    |
+| Argo                    | ✔️                | None (Login via Openshift [WIP](https://github.com/operate-first/odh/issues/39))   |
```

Comment:
Nit: The table columns look a little off :-)

Reply:
![image](https://user-images.githubusercontent.com/7453394/101037476-97440900-357b-11eb-8733-56a07e1ca4f7.png)

Are you sure it's not your git diff split view? :thinking:

Reply:
Hah, I see the problem. My :heavy_check_mark:  is showing up as smaller than the :x:  in the diff.